packages/kotti-ui/source/kotti-filters/types.ts

```diff
@@ -129,6 +129,7 @@ export namespace KottiFilters {
 			Common & {
 				operations: OPERATION[]
 				step: KottiFieldNumber.Props['step']
+				suffix: KottiFieldNumber.Props['suffix']
```

Comment:
the `prefix` seems to have already been supported but not part of this type

should you add `prefix` here as well?

Reply:
Float and Currency column types updated with prefix and suffix support.